src/questions/chapter5.yaml

```diff
@@ -0,0 +1,1075 @@
+      order: 2
+    - content: None
+      key: D
+      order: 3
+
+- uuid: 3c8b8711-b120-46db-9bc8-46d6fa292dae
+  type: SingleChoice
+  content: Which of the following inheritance does not belongs to solidity?
+  hint: noHint
+  explanation:
+  answerKeys:
+    - B
+  topic: Inheritance
+  subTopics:
+    - Inheritance types
+  difficultyLevel: Medium
+  choices:
+    - content: Single Inheritance
+      key: A
+      order: 0
+    - content: "Hybrid Inheritance: A combination of single, multiple inheritance and hierarchical inheritance."
+      key: B
+      order: 1
+    - content: "Hierarchical Inheritance: A parent has more than one child class/contract"
+      key: C
+      order: 2
+    - content: Multiple Inheritance
+      key: D
+      order: 3
+
+- uuid: 0d4494ae-27cf-40d7-bb43-071a9c11833a
+  type: SingleChoice
+  content: Which of the following explain about Multi-level Inheritance?
+  hint: noHint
+  explanation: Multi-level inheritance is similar to single inheritance, but it has levels of the relationship between the parent and the child. 
+  answerKeys:
+    - C
+  topic: Inheritance
+  subTopics: 
+    - Inheritance types
+  difficultyLevel: Low
+  choices:
+    - content: A single contract can be inherited from one or more contracts.
+      key: A
+      order: 0
+    - content: Single contract acts as a base contract for multiple derived contracts.
+      key: B
+      order: 1
+    - content: Multiple levels of the parent-child relationship.
+      key: C
+      order: 2
+    - content: None
+      key: D
+      order: 3
+
+- uuid: 672fcc59-0567-436f-8eaf-c43858082344
+  type: SingleChoice
+  content: Which of the following is a inbuilt library of solidity?
+  hint: noHint
+  explanation: Dapp-bin was created by Ethereum includes interesting and useful libraries like DoublyLinkedList, StringUtils, IterableMapping, etc.
+  answerKeys:
+    - A
+  topic: Libraries
+  subTopics:
+    - Inbuilt Library
+  difficultyLevel: Low
+  choices:
+    - content: Dapp-bin
+      key: A
+      order: 0
+    - content: Matplotlib
+      key: B
+      order: 1
+    - content: Seaborn
+      key: C
+      order: 2
+    - content: None
+      key: D
+      order: 3
+
+- uuid: ed3525c5-5e27-49b2-9975-7e421099e239
+  type: SingleChoice
+  content: Which of the below solidity keyword is used while importing library. 
+  hint: noHint
+  explanation: 
+    - "'for' keyword is used to attach library functions to any type."
+    -  "<libraryName> for <dataType>: libraryName is the name of the desired library to import, dataType is the variable type for which we want to access the library." 
+  answerKeys:
+    - B
+  topic: Library
+  subTopics:
+    - keyword
+  difficultyLevel: Low
+  choices:
+    - content: of
+      key: A
+      order: 0
+    - content: for
+      key: B
+      order: 1
+    - content: apply
+      key: C
+      order: 2
+    - content: match
+      key: D
+      order: 3
+
+- uuid: 2233d367-dd48-40d5-aeac-44a3a606578d
```

Comment:
Similar to another question related to library i.e. couple of question below this one